build.gradle

```diff
@@ -336,7 +336,7 @@ dependencyManagement {
 dependencies {
 
     implementation group: 'com.github.hmcts', name: 'et-common', version:'1.4.0'
-    implementation group: 'com.github.hmcts', name: 'et-data-model', version:'3.3.1'
+    implementation group: 'com.github.hmcts', name: 'et-data-model', version:'RET-3460-Final-SNAPSHOT'
```

Comment:
Update

Reply:
I think this needs to stay until it has been QA'd as they won't be able to verify it passes pipeline otherwise.